src/renderer/io/shortcuts.ts

```diff
@@ -15,8 +15,8 @@ import { Section } from "../../shared/ui/app";
 import { log } from "../logger";
 import { BrowserWindowEvent, IpcChannel } from "../../shared/ipc";
 
-const INITIAL_DELAY_MS = 400;
-const REPEAT_DELAY_MS = 200;
+const INITIAL_DELAY_MS = 300;
+const REPEAT_DELAY_MS = 150;
```

Comment:
Delays were reduced again because they had been previously raised in an attempt to fix this bug. 200 / 400 ms is too slow.